libraries/drupalSDK/src/main/java/com/ls/drupal/AbstractBaseDrupalEntity.java

```diff
@@ -35,49 +36,47 @@
 import com.ls.util.internal.ObjectComparator.Snapshot;
 import com.ls.util.internal.VolleyResponseUtils;
 
-import junit.framework.Assert;
-
-import android.support.annotation.NonNull;
-
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
 import java.util.HashMap;
 import java.util.Map;
 
+import junit.framework.Assert;
+
 public abstract class AbstractBaseDrupalEntity implements DrupalClient.OnResponseListener, ICharsetItem
 {
-	transient private DrupalClient drupalClient; 
+	transient private DrupalClient drupalClient;
 
 	transient private Snapshot snapshot;
 
 	/**
 	 * In case of request canceling - no method will be triggered.
```

Comment:
Dunno, it's in the drupal sdk, I'd ignore it and just get rid of the whole folder altogether 😅